sdk/identity/identity/samples/AzureIdentityExamples.md

```diff
@@ -431,6 +455,252 @@ function main() {
 }
 ```
 
+## Advanced Examples
+
+### Custom Credentials
+
+The `@azure/identity` library covers a broad range of Azure Active Directory authentication scenarios. However, it's possible the credential implementations provided might not meet the specific needs your application, or an application might want to avoid taking a dependency on the `@azure/identity` library.
+
+In this section we'll examine some example cases in which it might make sense to write a credential on your own.
```

Comment:
```suggestion
The `@azure/identity` library covers a broad range of Azure Active Directory authentication scenarios. However, it's possible the credential implementations provided might not meet the specific needs your application, or an application might want to avoid taking a dependency on the `@azure/identity` library. In such cases, you may want to write your own credential.

In this section we'll examine some such cases.
```